JSQSystemSoundPlayer/Classes/JSQSystemSoundPlayer.m

```diff
@@ -230,9 +236,11 @@ - (void)playAlertSoundWithFilename:(NSString *)filename fileExtension:(NSString
 
 - (void)playVibrateSound
 {
+#if TARGET_OS_IPHONE
```

Comment:
ahh... good point.

since it's unclear for now, let's go ahead and wrap the whole method and header. when/if APIs are released on OS X, then we can have a .1 update :smile: